packages/dashboard/src/features/database/pages/TablesPage.tsx

```
@@ -219,9 +225,12 @@ export default function TablesPage() {
     !!editingTable
   );
 
-  const primaryKeyColumn = useMemo(() => {
-    return schemaData?.columns.find((col) => col.isPrimaryKey)?.columnName;
-  }, [schemaData]);
+  // Full primary-key tuple (supports composite keys) used for row identity,
+  // updates, and deletes.
+  const primaryKeyColumns = useMemo(
+    () => getPrimaryKeyColumns(schemaData?.columns),
+    [schemaData]
+  );
```

ESLint

[error] Replace `⏎····()·=>·getPrimaryKeyColumns(schemaData?.columns),⏎····[schemaData]⏎··` with `()·=>·getPrimaryKeyColumns(schemaData?.columns),·[schemaData]`

(prettier/prettier)